proto/vtctldata.proto

```diff
@@ -1230,6 +1231,17 @@ message UpdateCellsAliasResponse {
   topodata.CellsAlias cells_alias = 2;
 }
 
+message WorkflowUpdateRequest {
+  string keyspace = 1;
+  // TabletRequest gets passed on to each primary tablet involved
+  // in the workflow via the UpdateVRWorkflow tabletmanager RPC.
+  tabletmanagerdata.UpdateVRWorkflowRequest tablet_request = 2;
+}
+
+message WorkflowUpdateResponse {
+  string results = 1;
```

Comment:
Instead of consolidating on the server-side (and then forcing clients to then un-consolidate if they want to do anything with those results):

```suggestion
  repeated string results = 1;
```

Reply:
Yeah, I want to think about how to improve the results.